bindings/nodejs/lib/Account.ts

```diff
@@ -441,7 +442,9 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
-    async getParticipationEvents(): Promise<ParticipationEvent[]> {
+    async getParticipationEvents(): Promise<{
+        [eventId: ParticipationEventId]: ParticipationEventWithNodes;
+    }> {
```

Comment:
This is correct, yeah ? I have no clue but it's so weird

Reply:
It's correct, it symbols a dictionary with eventId as a key and ParticipationEventWithNodes as a value.